### ISSUE TITLE
Improve the example on READMe.rst

### ISSUE THREAD

**Issue by dsouzadaniel** [ericmjl/pyjanitor#179]
I'd like to propose a change to the README.rst example which shows the difference between using regular Pandas and PyJanitor to achieve different operations.

The 1st line itself is something like :
df = pd.DataFrame(...)  # create a pandas DataFrame somehow.

For a newbie, who basically copy pastes this example, it'll just fail. 

I'd like to propose adding a better example with "before" and "after" tables to explain how PyJanitor makes this stuff easy! :) 

Thoughts?! :)

**Comment by dave-frazzetto** [comment]
I agree. Also, the same applies to the docs in functions.py. Some docs have associated runnable examples, others don't. Going through each with runnable examples would make sure that they are correct.

**Comment by dsouzadaniel** [comment]
Feel free to raise other documentation issues ( so others can also work on them! ) 😄
